packages/badge/src/badge.css

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+@import './spectrum-badge.css';
+
+/* center align icons and text */
+
+:host {
+    display: inline-flex;
+    align-items: center;
+}
+
+/* disable :hover until Spectrum CSS disables hover upstream */
+
+:host(:hover) {
+    background-color: var(--spectrum-badge-m-textonly-background-color);
+    color: var(--spectrum-badge-m-texticon-text-color);
+}
+
+/* implement fixed placement styling until Spectrum CSS adds it upstream */
+
+:host([fixed='top']) {
+    --spectrum-badge-neutral-textonly-border-top-left-radius: 0;
+    --spectrum-badge-neutral-textonly-border-top-right-radius: 0;
+}
+
+:host([fixed='right']) {
+    --spectrum-badge-neutral-textonly-border-top-right-radius: 0;
+    --spectrum-badge-neutral-textonly-border-bottom-right-radius: 0;
+}
+
+:host([fixed='bottom']) {
+    --spectrum-badge-neutral-textonly-border-bottom-left-radius: 0;
+    --spectrum-badge-neutral-textonly-border-bottom-right-radius: 0;
+}
+
+:host([fixed='left']) {
+    --spectrum-badge-neutral-textonly-border-top-left-radius: 0;
+    --spectrum-badge-neutral-textonly-border-bottom-left-radius: 0;
+}
+
+/* cascade badge's size to its icon */
```

Comment:
Doesn't need to block, but we've added this in a couple of places, might be useful to share a single CSS import rather that copy/paste.